src/ctl.c

```diff
@@ -97,6 +97,7 @@ CTL_PROTO(opt_hpa)
 CTL_PROTO(opt_hpa_slab_max_alloc)
 CTL_PROTO(opt_hpa_hugification_threshold)
 CTL_PROTO(opt_hpa_hugify_delay_ms)
+CTL_PROTO(opt_hpa_min_purge_interval_ms)
```

Comment:
Nit: We should add this to stats output, too. (Broadly, this is true for any new config option; it should go in jemalloc.c opt parsing, in the ctl opt namespace, and in the stats opt output).

Reply:
Thanks for letting me know, will add it there.